news/2021-07-18-mca-ayim-2020.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Mapper's Choice Awards and A Year In Mapping 2019"
+date: 2021-07-18 00:12:00 +0000
+---
+
+Hot of the tails of the previous edition, mappers can now celebrate 2020 achievements in *Mappers' Choice Awards* (MCA) AND *A Year In Mapping* (AYIM) as well as enjoy 2019's results!
+
+![](/wiki/shared/news/2021-07-17-ayim/mca-ayim-2020.png)
+
+After their grand return just [last month](https://osu.ppy.sh/home/news/2021-06-15-mca-ayim-2019) these two popular events are now here to catch up with the present! With newly revised categories, website quality-of-life updates, and badges now being given to the winner for every category, the scale of the events are only growing from here. Most events are by mappers, for mappers, but *everyone* is welcome to check out the festivities and leave comments on mappers they like from 2020!
+
+## Schedule for MCA and AYIM 2020
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Nomination Stage | 2021-07-18 12:00 UTC - 2021-07-23 00:00 UTC |
+| Voting Stage | 2021-07-24 00:00 UTC - 2021-08-04 00:00 UTC | 
+| Results Stream | 2021-08-07 / 2021-08-08 |
+| Results posted to Website | 2021-08-09 00:00 UTC
+
+## Mapper's Choice Awards 2020
+
+Hopefully our recent edition has refreshed everyone on what MCA is. If not: it's a voting event with multiple categories, where mappers can nominate and then vote on what they think is the best in each given category, as well as what are the best maps of the year in the Grand Award. On top of that, we also have categories for voting on the generally best mappers, most promising newcomers, and more. The categories have been newly revised for our 2020 event as well.
+
+If you ranked any beatmap or storyboard during 2020, you're now invited to help the mapping community decide those best of the best maps.
+
+As in the 2019 edition, the Mapper's Choice Awards has three stages: nomination, voting, and results. Be sure to check our schedule above to know when each stage happens!
+
+- **Nomination:** Eligible users can submit any beatmaps/storyboards or users from 2020 they feel are deserving of recognition in their respective categories.
+- **Voting:** Eligible users can vote on the pool of submitted nominations.
+- **Results:** Will be based off the results of voting and feature a livestream discussing each of the top winners. 
+
+Those who win regular awards on their beatmaps or storyboards in the results will be given 2 months of supporter, and grand prize winners will receive 6 months of supporter. In addition, those who place first in any category will receive a unique **MCA 2020 badge** for their achievement. If you're not eligible to vote in MCA, your reward for following along will be getting to find out about super snazzy 2020 maps.
```

Comment:
`Grand Award winners will receive six months of supporter, while winners of other categories will receive two months of supporter instead. Additionally, all of the category winners will receive a unique **MCA 2020 profile badge** for their achievement. `